From ecc9a52cfe944e72f7539917e0b2df2e4b8219c9 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Mon, 8 Jul 2013 15:22:05 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 keystoneclient/__init__.py | 4 +---
 setup.py                   | 1 -
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/keystoneclient/__init__.py b/keystoneclient/__init__.py
index bd50f9f..991e8e6 100644
--- a/keystoneclient/__init__.py
+++ b/keystoneclient/__init__.py
@@ -13,8 +13,6 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import pbr.version
-
 from keystoneclient import access
 from keystoneclient import client
 from keystoneclient import exceptions
@@ -25,7 +23,7 @@ from keystoneclient import v2_0
 from keystoneclient import v3
 
 
-__version__ = pbr.version.VersionInfo('python-keystoneclient').version_string()
+__version__ = "REDHATKEYSTONECLIENTVERSION"
 
 __all__ = [
     # Modules
diff --git a/setup.py b/setup.py
index 7363757..09230a0 100644
--- a/setup.py
+++ b/setup.py
@@ -26,5 +26,4 @@ except ImportError:
     pass
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
-- 
1.9.3

